api/test/helpers/auth.js

```diff
@@ -0,0 +1,40 @@
+const { parse, stringify } = require('querystring');
+const request = require('supertest');
+const { URL } = require('url');
+
+const buildCookie = (setCookie) =>
+  setCookie.map((cookieEntry) => cookieEntry.split(';')[0]).join('; ');
+
+module.exports = {
+  implicitFlowLogin: async (server, login, password, client) => {
```

Comment:
I originally tried the authorization flow, then switched to implicit flow while troubleshooting other issues. I opted to keep this since it was pretty much done, but if we should do auth flow instead it's probably a simple change back.